### DISCUSSION TITLE
Descontos Financeiros no Saft

### DISCUSSION THREAD

**Discussion by GuFox** [assoft-portugal/SAF-T-AO#207] [Q&A] [answered]
Boa tarde pessoal alguém tem uma dica como posso representar os descontos financeiros no SAFT ?

**Comment by cryptolopes** [comment, marked as answer]
### DocumentTotals
https://github.com/assoft-portugal/SAF-T-AO/blob/d8c3af82ae5ed113cd29c3c0ba2400b554e5e7b1/XSD/SAFTAO1.01_01.xsd#L556

### Settlement
https://github.com/assoft-portugal/SAF-T-AO/blob/d8c3af82ae5ed113cd29c3c0ba2400b554e5e7b1/XSD/SAFTAO1.01_01.xsd#L1045-L1081
